.release-please-manifest.json

```diff
@@ -0,0 +1 @@
+{".":"0.4.1","acir":"0.4.1","acir_field":"0.4.1","acvm":"0.4.1","stdlib":"0.4.1"}
```

Comment:
This gets updated by release-please when a breaking change is added via a commit?

Reply:
It's updated by release-please for any version bumps. 🎉

Reply:
Oh nice!